### PR TITLE
(BKR-647) windows.install_package now uses SSL cygwin URL with fallback

### PR DESCRIPTION
**Pull request by kevpl**

### CODE REVIEW

**Comment by rick** [comment]
:+1:

**Comment by kevpl** [comment]
@puppetlabs-jenkins retest this please

File: lib/beaker/host/windows/pkg.rb [hunk 1]
```diff
@@ -26,7 +26,13 @@ def install_package(name, cmdline_args = '')
     end
 
     if not check_for_command(cygwin)
-      execute("curl --retry 5 http://cygwin.com/#{cygwin} -o /cygdrive/c/Windows/System32/#{cygwin}")
+      begin
+        command = "curl --retry 5 https://cygwin.com/#{cygwin} -o /cygdrive/c/Windows/System32/#{cygwin}"
+        execute(command)
+      rescue Beaker::Host::CommandFailure
+        command.sub!('https', 'http')
```
Comment (colinPL) [inline lib/beaker/host/windows/pkg.rb]: Is `command` defined here? The variable is set under `begin`.

File: lib/beaker/host/windows/pkg.rb [hunk 2]
```diff
@@ -26,7 +26,13 @@ def install_package(name, cmdline_args = '')
     end
 
     if not check_for_command(cygwin)
-      execute("curl --retry 5 http://cygwin.com/#{cygwin} -o /cygdrive/c/Windows/System32/#{cygwin}")
+      begin
+        command = "curl --retry 5 https://cygwin.com/#{cygwin} -o /cygdrive/c/Windows/System32/#{cygwin}"
```
Comment (colinPL) [inline lib/beaker/host/windows/pkg.rb]: This `command =` can go above the `begin` statement.

**Pull request merged**